### ISSUE TITLE
Code review

### ISSUE THREAD

**Issue by aquastripe** [sayksii/Plant_Seedlings_Recognition#1]
- 取出預測結果的方式
  ```py
  _, preds = torch.max(outputs, 1)
  ```
  改成
  ```py
  preds = torch.argmax(outputs, 1)
  ```
  會更直白、簡潔
- 儲存模型的時候，推薦優先選擇最低 validation loss/最高 validation accuracy 而非直接存最後一次訓練結果

整體來說寫得很好，沒什麼問題
